LICENSE

```diff
@@ -1,21 +1,29 @@
-The MIT License (MIT)
+BSD 3-Clause License
 
-Copyright (c) 2015, The Jupyter Development Team
+Copyright (c) 2017, Project Jupyter Contributors
```

Comment:
Also please don't change the year. A single year is implicitely YYYY-Present, changing the year would basically be saying we relinquish our copyright for the 2015-2017 period.